cmd/ef_tests/levm/deserialize.rs

```diff
@@ -122,103 +122,104 @@ where
-                    let tx = EFTestTransaction {
-                        data: data.clone(),
-                        gas_limit: *gas_limit,
-                        gas_price: raw_tx.gas_price,
-                        nonce: raw_tx.nonce,
-                        secret_key: raw_tx.secret_key,
-                        sender: raw_tx.sender,
-                        to: raw_tx.to.clone(),
-                        value: *value,
-                    };
-                    transactions.push(((data_id, gas_limit_id, value_id), tx));
-                }
-            }
-        }
-
-        let ef_test = Self {
-            name: test_name.to_owned().to_owned(),
-            _info: serde_json::from_value(
-                test_data
-                    .get("_info")
-                    .ok_or(serde::de::Error::missing_field("_info"))?
-                    .clone(),
-            )
-            .map_err(|err| {
-                serde::de::Error::custom(format!(
-                    "error deserializing test \"{test_name}\", \"_info\" field: {err}"
-                ))
-            })?,
-            env: serde_json::from_value(
-                test_data
-                    .get("env")
-                    .ok_or(serde::de::Error::missing_field("env"))?
-                    .clone(),
-            )
-            .map_err(|err| {
-                serde::de::Error::custom(format!(
-                    "error deserializing test \"{test_name}\", \"env\" field: {err}"
-                ))
-            })?,
-            post: serde_json::from_value(
-                test_data
-                    .get("post")
-                    .ok_or(serde::de::Error::missing_field("post"))?
-                    .clone(),
-            )
-            .map_err(|err| {
-                serde::de::Error::custom(format!(
-                    "error deserializing test \"{test_name}\", \"post\" field: {err}"
-                ))
-            })?,
-            pre: serde_json::from_value(
+            let raw_tx: EFTestRawTransaction = serde_json::from_value(
                 test_data
-                    .get("pre")
-                    .ok_or(serde::de::Error::missing_field("pre"))?
+                    .get("transaction")
+                    .ok_or(serde::de::Error::missing_field("transaction"))?
                     .clone(),
             )
             .map_err(|err| {
                 serde::de::Error::custom(format!(
-                    "error deserializing test \"{test_name}\", \"pre\" field: {err}"
+                    "error deserializing test \"{test_name}\", \"transaction\" field: {err}"
                 ))
-            })?,
-            transactions,
-        };
-        Ok(ef_test)
+            })?;
+
+            let mut transactions = HashMap::new();
+
+            // Note that inthis order of iteration, in an example tx with 2 datas, 2 gasLimit and 2 values, order would be
+            // 111, 112, 121, 122, 211, 212, 221, 222
```

Comment:
I think we can delete this comment since the `tx_id` notion is not used anymore

Reply:
Yes, let's remove it in a cleanup PR